grassroot-services/src/main/java/za/org/grassroot/services/task/EventBrokerImpl.java

```diff
@@ -933,6 +935,31 @@ public String getMostFrequentLocation(String groupUid) {
 		return eventRepository.findAll(specifications, sort);
 	}
 
+	@Override
+	@Transactional(readOnly = true)
+	@SuppressWarnings("unchecked")
+	public String getMostFrequentEventTime(String groupUid){
+		Group group = groupRepository.findOneByUid(groupUid);
 
+		String qry = "SELECT to_char(e.eventStartDateTime,'hh24:mi') as event_time, " +
+				"COUNT(e.eventStartDateTime) as count_time " +
```

Comment:
This logic won't work. Instead get the raw strings and then retrieve the most frequent. See:

https://stackoverflow.com/questions/22989806/find-the-most-common-string-in-arraylist